grr/server/grr_response_server/gui/ui/components/flow_details/plugins/osquery_details.ng.html

```diff
@@ -12,12 +12,12 @@
     <div class="table-container">
       <table>
         <thead>
-          <th *ngFor="let column of resultTable.header.columns">{{column.name}}</th>
+          <th *ngFor="let column of resultTable.header.columns; trackBy: trackByIndex">{{column.name}}</th>
         </thead>
         <tbody>
-          <tr *ngFor="let rowValues of resultTable.rows">
-            <td *ngFor="let value of rowValues.values">
-              {{value}}
+          <tr *ngFor="let row of resultTable.rows; trackBy: trackByIndex">
+            <td *ngFor="let cellValue of row.values; trackBy: trackByIndex">
```

Comment:
`trackBy` is not needed for primitives (which I assume `cellValue` is because you print it?)